tsconfig.json

```diff
@@ -4,6 +4,7 @@
     "strict": true,
     "noUncheckedIndexedAccess": true,
     "forceConsistentCasingInFileNames": true,
+    "exactOptionalPropertyTypes": true,
```

Comment:
I'll add try to add this to Airnode as well.

Reply:
Nice - this looks like a great option to have enabled

Reply:
https://api3dao.atlassian.net/browse/AN-562